compact_str/src/repr/mod.rs

```diff
@@ -311,6 +314,10 @@ impl Repr {
 
     #[inline]
     pub fn push_str(&mut self, s: &str) {
+        if s.is_empty() {
+            return;
+        }
+
```

Comment:
Why was this case added? Can you add a comment explaining

Reply:
This is for cases where `s` is empty, there's no need to reserve anything in that case, hence I added this.